db-service/lib/common/DatabaseService.js

```diff
@@ -35,6 +35,21 @@ class DatabaseService extends cds.Service {
     throw '2b overridden by subclass'
   }
 
+  init() {
+    this.on(['*'], this.onRequest)
```

Comment:
fyi, you could use [prepend](https://cap.cloud.sap/docs/node.js/core-services#srv-prepend) to register before others and, hence, keep `return super.init()` in SQLService